internal/storage/elasticsearch/mapping/command_test.go

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2025 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package mapping
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/jaegertracing/jaeger/cmd/esmapping-generator/app"
+	"github.com/jaegertracing/jaeger/pkg/testutils"
+)
+
+func TestCommandExecute(t *testing.T) {
+	o := app.Options{
+		Mapping:       "jaeger-span",
+		EsVersion:     7,
+		Shards:        5,
+		Replicas:      1,
+		IndexPrefix:   "jaeger-index",
+		UseILM:        "false",
+		ILMPolicyName: "jaeger-ilm-policy",
+	}
+	cmd := Command(o)
+
+	oldStdout := os.Stdout
+	defer func() {
+		os.Stdout = oldStdout
+	}()
+
+	r, w, err := os.Pipe()
```

Comment:
You can use testify TempFile capability and os.ReadAll, it will be less code that a pipe.